db/db.sql

```diff
@@ -41,6 +41,18 @@ CREATE TABLE `job_errors` (
   `jobID` char(36) not null,
   `url` varchar(4096) not null,
   `message` varchar(4096) not null,
+  `level` varchar(255) check (
+  `level` in (
+    'error',
+    'warning'
+  )
+  ) not null default 'error',
+  `category` varchar(255) check (
+    `category` in (
```

Comment:
Do we want to have to change this check every time a new category is added in service lib? In theory, if we're already adding the category via pull request in service lib, it has been validated already by the time it gets to Harmony. This is also missing the existing error categories like "Server", "Forbidden".

Reply:
That's a good point. I'll drop this constraint here as well as in the migration.